src/gleam/dynamic.gleam

```diff
@@ -27,33 +27,45 @@ pub external fn from(a) -> Dynamic =
 pub external fn unsafe_coerce(Dynamic) -> a =
   "gleam_stdlib" "identity"
 
-/// Check to see whether a Dynamic value is a bit_string, and return the bit_string if
+external fn erl_string(from: Dynamic) -> Result(BitString, String) =
```

Comment:
Could we remove this and use the `bit_string` function below?

Reply:
Updated